tarok/test/tarok_utils.cpp

```diff
@@ -28,4 +28,13 @@ bool AllActionsInOtherActions(
   return true;
 }
 
+open_spiel::Action ActionFromCardName(std::string long_name,
```

Comment:
Not really important but should probably be named with the same convention as some existing similar methods like `ActionToCard`, e.g. `CardNameToAction`.

Reply:
Change to CardLongNameToAction.